<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageSetupClass</class>
 <widget class="QDialog" name="PageSetupClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>522</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyWhiteboard - Printing Page Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbPaper">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Screen Resolution</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="cbScreenResolution">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="placeholderText">
         <string>Select Screen Resolution</string>
        </property>
        <item>
         <property name="text">
          <string>2840 x 2160 (16:9 - 4K)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3440 x 1440 (21:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2560 x 1440 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2560 x 1080 (21:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2048 x 1152 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920 x 1200 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1920 x 1080 (16:9 - Full HD)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1680 x 1050 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1600 x 900 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1536 x 864 (16 x 9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1440 x 900 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1366:768 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1360 x 768 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 1024 (5:4)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 800 (16:10)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1280 x 720 (16:9)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1024 x 768 (4:3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800 x 600 (4:3)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>640 x 360 (16:9</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>107</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>or Horizontal Pixel Count to print on one page</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="sbHorizPixels">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="suffix">
         <string> pixels</string>
        </property>
        <property name="minimum">
         <number>300</number>
        </property>
        <property name="maximum">
         <number>16355</number>
        </property>
        <property name="singleStep">
         <number>11</number>
        </property>
        <property name="value">
         <number>1920</number>
        </property>
        <property name="displayIntegerBase">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Screen diagonal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="edtScreenDiag">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>24</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="cbUnit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>inch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>mm</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Printer</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Paper Orientation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cbOrientation">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Portrait</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Landscape</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkWhiteBackground">
           <property name="text">
            <string>&amp;White background</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="chkGrayscale">
           <property name="text">
            <string>&amp;Grayscale</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkPrintBackgroundImage">
           <property name="text">
            <string>Print &amp;Background Image</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="chkGrid">
           <property name="text">
            <string>G&amp;rid</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="cbPrinterSelect">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Select Printer</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLabel" name="lblPrinter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string> --- not selected ---</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>PageSetupClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PageSetupClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
